python/pyspark/pandas/__init__.py

```diff
@@ -151,3 +154,13 @@ def _auto_patch_pandas() -> None:
 from pyspark.pandas.config import get_option, options, option_context, reset_option, set_option
 from pyspark.pandas.namespace import *  # noqa: F403
 from pyspark.pandas.sql_formatter import sql
+
+
+def __getattr__(key: str) -> Any:
+    if key.startswith("__"):
+        raise AttributeError(key)
+    if hasattr(_MissingPandasLikeGeneralFunctions, key):
+        func = getattr(_MissingPandasLikeGeneralFunctions, key)
+        return partial(func)
```

Comment:
May I ask why not return `func` directly?

Reply:
Oh, in this case we don't need to `partial`. Thanks!